### ISSUE TITLE
Courgette/zuchinni waffles

### ISSUE THREAD

**Issue by iteles** [iteles/blw-baby#20]
https://thatvegannephew.com/zucchini-waffles-chickpea-flour/

@Stephanymtr Let's make a half dose of this to try?

Ingredients for half a dose:
+ [ ] 1.25 cups grated zucchini
+ [ ] 0.5 cups chickpea flour
+ [ ] 0.25 cup all-purpose flour **[I suggest we try with wholewheat flour]**
+ [ ] 1.75  teaspoons baking powder
+ [ ] 0.75 tablespoon nutritional yeast
+ [ ] 1/8 teaspoon smoked paprika (original recipe called for 0.25)
+ [ ] 1/8 teaspoon garlic powder (original recipe called for 0.25)
+ [ ] 0.5 cup unsweetened soy milk, or your favorite plant-based milk **[I suggest we use oat milk]**
+ [ ] 1 tablespoons olive oil
+ [ ] 0.5 tablespoon ground flaxseed

Grate the zucchini, until you have about 2 ½ cups packed. Set this aside.
Combine together the chickpea and all-purpose flours, baking powder, nutritional yeast, salt, smoked paprika and garlic powder in a large mixing bowl.
Then in a separate bowl, whisk together the soy milk, olive oil and ground flaxseed until well incorporated.
Add the soy milk mixture to the dry ingredients, then add the grated zucchini. Fold everything together with a spatula until it becomes a smooth batter with no dry flour remaining.
Let the batter rest for 5 minutes, and heat up your waffle iron in the meantime.
Take a heaping ½ cup of batter, and place on the center of the waffle iron. My waffles took about 3 – 4 minutes to produce a perfect golden brown waffle, but it depends on the kind of iron you have.
Serve with vegan butter, or even Follow Your Heart parmesan cheese and/or hot sauce!

**Comment by Stephanymtr** [comment]
This was a successful family friendly recipe. I used a basting brush with melted butter to grease the pre heated waffle maker and left the batter in for 4 minutes.

**Comment by iteles** [comment]
Agree, these worked great! They are better fresh - baby wasn't too interested in them after they'd been in the fridge.

I reckon next time we freeze a couple straight away and freeze some batter straight away and see whether either use is successful.

I would also add grated carrots (maybe sub out part of the courgette for them) next time for added interest in the flavour.
